contracts/BondStorage.sol

```diff
@@ -216,7 +228,7 @@ contract BondStorage is AccessControl {
         uint256 _maturityInWeeks,
         uint256 _tokenId,
         uint128 _liquidity
-    ) external {
+    ) external onlyWhitelisted {
```

Comment:
did a fix earlier to make sure this function is callable by any user, see https://github.com/the-standard/ibco/pull/75. so we should not protect it.

Reply:
that's confusing to me
so 1. users can still create bonds when we've paused stage 2 through the operator?
2. a user can provide their own rate, maturity etc?

Reply:
thought we were at the operator, my bad, and good spot for this severe bug!